includes/course-theme/class-sensei-course-theme-lesson.php

```diff
@@ -116,8 +116,14 @@ private function maybe_add_quiz_results_notice() {
 	 * Maybe add lesson prerequisite notice.
 	 */
 	private function maybe_add_prerequisite_notice() {
+		$lesson_id = \Sensei_Utils::get_current_lesson();
```

Comment:
It fixes a case to not show the prerequisite notice if the user is not enrolled.